kafka/src/main/java/zipkin2/reporter/kafka/KafkaSender.java

```diff
@@ -201,20 +250,39 @@ public KafkaSender build() {
   }
 
   final Properties properties;
+  final Properties adminClientProperties;
   final String topic;
   final Encoding encoding;
   final BytesMessageEncoder encoder;
   final int messageMaxBytes;
 
   KafkaSender(Builder builder) {
     properties = new Properties();
+    adminClientProperties = new Properties();
     properties.putAll(builder.properties);
+    adminClientProperties.putAll(filterPropertiesForAdminClient(properties));
+    adminClientProperties.putAll(builder.adminClientProperties);
     topic = builder.topic;
     encoding = builder.encoding;
     encoder = BytesMessageEncoder.forEncoding(builder.encoding);
     messageMaxBytes = builder.messageMaxBytes;
   }
 
+  /**
+   * Filter the properties configured for the producer by removing those not used for the Admin Client.
+   *
+   * See @{@link AdminClientConfig} config properties
+   */
+  private Map<String, Object> filterPropertiesForAdminClient(Properties properties){
```

Comment:
I think the user should be responsible for only putting valid properties. note we dont do the other way..